### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by sss1612**
A portion of the documentation was slightly contradictory, hopefully this makes more sense

### CODE REVIEW

**Comment by intcreator** [comment]
thanks for your PR! unfortunately I believe your change to the documentation is incorrect. by default jobs are _not_ automatically started. the `start` param is `false` by default.

I can see how this note can be a little confusing since it could be more clear that it's only referring to the example, but adding clarification about how it only refers to the example is all I would change

File: README.md [hunk 1]
```diff
@@ -39,11 +39,10 @@ var job = new CronJob(
     true,
     'America/Los_Angeles'
 );
-// Use this if the 4th param is default value(false)
-// job.start()
+// job.start() - See note below when to use this
 ``` 
 
-Note - You don't need to explicitly start a job in order to make it run since the 4th parameter is set to `true`. However, by default you need to call `job.start()` to start the cron job, which gives a little more control over running your jobs. 
+Note - By default, jobs are started automatically after constructing a `CronJob()` object. You can turn this feature off by setting the 4th `CronJob()` parameter to `false` so job(s) can be started when needed.
```
Comment (sss1612) [inline README.md:45]: @intcreator Oof, my apologies! I'm not sure how I messed up my interpretation this badly. Would this be better/acceptable?
```suggestion
Note - In the above example, the 4th parameter of `CronJob` automatically starts the job upon creation. By default, jobs need to be programmatically started using `CronJob.start()` if this parameter is not specified/falsy.
```

**Comment by intcreator** [comment]
I made a couple of tweaks to the wording but other than that, looks good! merging this now

**Pull request merged**